spec/features/user_votes_on_review_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+feature "users can add a new review for a piece of music" do
+  let(:user) { FactoryGirl.create(:user) }
+  let(:piece) { FactoryGirl.create(:piece, user: user) }
+  let!(:review) { FactoryGirl.create(:review, user: user, piece: piece) }
+
+  context "not logged in user" do
+    scenario "cannot vote" do
+      visit piece_path(piece)
+      click_link '+'
+
+      expect(page).to have_content "You need to sign in or sign up before continuing"
+    end
+  end
+
+  context "logged in user" do
+    before do
+      sign_in_as(user)
+    end
+
+    scenario "user sees upvote and downvote" do
+      visit piece_path(piece)
+      expect(page).to have_content("+")
+      expect(page).to have_content("-")
+    end
```

Comment:
Delete this scenario, it seems like no logged in users can see this content as well. Plus, you will test this once you actually start clicking on the links.